screenshots/build.js

```diff
@@ -23,6 +23,7 @@ page.open('./test/perceptual/typography.html', function() {
 	setTimeout(function() {
 			capture('paragraph');
 			capture('headings', 200);
+			capture('body-2');
```

Comment:
This looks to possibly be a combination of the screenshot tool and committing the screenshot. I compared the demo site (which looks as expected) with the screenshots and all look slightly darker. It seems when the screenshots are committed they become blurry. These things combined make the screenshot look more bold than it should.